src/Parsing/Common/SummaryHelper.php

```diff
@@ -8,19 +8,27 @@
 class SummaryHelper
 {
     /**
-     * Adds custom separators for console display in line-items-like fields.
+     * Formats decimal numbers for display, by removing all excess digits after the fifth.
      *
-     * @param array  $columnSizes Sizes of the respective columns.
-     * @param string $separator   Separator character.
-     * @return string
+     * @param float|null $number Number to parse.
+     * @return string Number as a valid float string.
      */
-    public static function lineSeparator(array $columnSizes, string $separator): string
+    public static function formatFloat(?float $number): string
     {
-        $outStr = "  +";
-        foreach ($columnSizes as $size) {
-            $outStr .= str_repeat($separator . "+", $size);
+        if ($number === null) {
+            return '';
+        }
+        $formatted = number_format($number, 5, '.', '');
+
+        $formatted = rtrim(rtrim($formatted, '0'), '.');
+
+        if (substr_count($formatted, '.') == 0) {
+            $formatted .= '.00';
+        } elseif (strlen($formatted) - strpos($formatted, '.') <= 2) {
+            $formatted = sprintf("%.2f", $formatted);
```

Comment:
I'm so sorry

Reply:
It's truly fascinating how blunt of a tool PHP is at times